keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c

```diff
@@ -0,0 +1,117 @@
+/* Copyright 2018 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define FN 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[BASE] = LAYOUT(
+		KC_7, KC_8, KC_9, \
+		KC_4, KC_5, KC_6, \
+		KC_1, KC_2, KC_3, \
+		MO(1), KC_0, KC_DOT \
+	),
+  
+	[FN] = LAYOUT(
+		KC_TRNS, KC_HOME, KC_PGUP, \
+		KC_TRNS, KC_END, KC_PGDN, \
+		KC_TRNS, KC_TRNS, KC_TRNS, \
+		KC_TRNS, KC_TRNS, KC_ENT \
+	)
+};
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_WH_L);
```

Comment:
If you're using the mouse keys here, you may need to add `#define TAP_CODE_DELAY 10` to the config.h for the keymap.   

Though, if it works just fine, then ignore.